docs/rules/bugs/var-shadows-builtin.md

```diff
@@ -0,0 +1,61 @@
+# var-shadows-builtin
+
+**Summary**: Variable shadows built-in
+
+**Category**: Bugs
+
+**Avoid**
+```rego
+package policy
+
+import rego.v1
+
+# variable `http` shadows `http.send` built-in function
+allow if {
+    http := true
```

Comment:
Perhaps `http := startswith(url, "http://")` or similar, just to make it more real feeling?

Reply:
Alright!